docs/API.md

```diff
@@ -831,7 +870,7 @@ if err != nil {
 }
 ```
 <a name="StatObject"></a>
-### StatObject(bucketName, objectName string) (ObjectInfo, error)
+### StatObject(bucketName, objectName string, opts GetObjectOptions) (ObjectInfo, error)
```

Comment:
It looks odd to pass GetObjectOptions to a HEAD call.

Reply:
The problem is a SSE-C call requires encryption keys - Otherwise there would be no way to get stats of an SSE-C object...

Reply:
Perhaps `StatObject(bucketName, objectName string, opts StatObjectOptions)`  --> `GetObjectOptions --> StatObjectOptions`